tasks/harmonic-series/src/main.rs

```diff
@@ -0,0 +1,78 @@
+use num::rational::Ratio;
+use num::BigInt;
+
+fn main() {
+    for n in 1..=20 {
+        println!("Harmonic number {n} = {}", harmonic_number(n));
+    }
+
+    println!("Harmonic number 100 = {}", harmonic_number(100));
+
+    //In order to avoid recomputing all the terms in the sum for every harmonic number
```

Comment:
Typically we leave a space between the comment marker and the comment: `// In` vs `//In`.

Reply:
Thank you for the feedback! I have added a space to the beginning of every comment line to separate comment markers and the text.